pinc/project_quick_check.inc

```diff
@@ -266,27 +256,80 @@ function _test_project_for_corrupt_pngs($projectid)
     return ["name" => $test_name, "description" => $test_desc, "status" => $status, "summary" => $summary, "details" => $details];
 }
 
+function _test_project_for_small_images($projectid)
+{
+    $test_name = _("Small page images");
+    $test_desc = _("This test checks to see if any page image is possibly too small.");
```

Comment:
Subject and verb are in agreement, so I'm okay with being shouted down, but it makes more sense to me to say "if any page images are possibly too small", because the check is performed on all page images.